src/main/java/pro/sky2/HW5/Set/EmployeeController1.java

```diff
@@ -0,0 +1,37 @@
+package pro.sky2.HW5.Set;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+@RestController
+public class EmployeeController1 {
+    private final EmployeeService1 employeeService1;
+
+
+    public EmployeeController1(EmployeeService1 employeeService1) {
+        this.employeeService1 = employeeService1;
+    }
+
+    @GetMapping(path = "/post/employee-list/")
```

Comment:
путь до конечных точек лучше инвертировать: сначала сущность, потом действие
```suggestion
    @GetMapping(path = "/employee-list/post")
```